### ISSUE TITLE
Null Pointer Exception on DeviceInfo.java on sdk version 3.5.1

### ISSUE THREAD

**Issue by Aazu07** [CleverTap/clevertap-android-sdk#33]
The Logs for the crash is as:

```
Caused by java.lang.NullPointerExceptionAttempt to invoke virtual method 'void c.g.a.a.ac.e(java.lang.String, java.lang.String)' on a null object reference Raw Text
--
  | com.clevertap.android.sdk.DeviceInfo.initDeviceID + 98 (DeviceInfo.java:98)
  | com.clevertap.android.sdk.DeviceInfo.<init> + 54 (DeviceInfo.java:54)
  | com.clevertap.android.sdk.CleverTapAPI.<init> + 241 (CleverTapAPI.java:241)
  | com.clevertap.android.sdk.CleverTapAPI.instanceWithConfig + 565 (CleverTapAPI.java:565)
  | com.clevertap.android.sdk.CleverTapAPI.instanceWithConfig + 542 (CleverTapAPI.java:542)
  | com.clevertap.android.sdk.CTInAppBasePartialFragment.generateListener + 25 (CTInAppBasePartialFragment.java:25)
  | com.clevertap.android.sdk.CTInAppBaseFragment.onAttach + 63 (CTInAppBaseFragment.java:63)
  | android.app.Fragment.onAttach + 1484 (Fragment.java:1484)
  | androidx.appcompat.app.AppCompatActivity.onCreate + 85 (AppCompatActivity.java:85)
  | com.railyatri.in.mobile.BaseParentActivity.onCreate + 77 (BaseParentActivity.java:77)

```

**Comment by darshanclevertap** [comment]
@Aazu07 Thanks for pointing this out. This has been fixed for the upcoming release. I will keep you informed when the release is done.

**Comment by darshanclevertap** [comment]
@Aazu07 This has been fixed in v3.6.0. Closing this issue.
